src/pages/LandingPage.tsx

```diff
@@ -12,6 +12,14 @@ import { ApiState } from "../kusama/KusamaContext"
 import KappaSigmaMuTitle from '../static/kappa-sigma-mu-title.svg'
 import { LoadingSpinner } from "./explore/components/LoadingSpinner"
 
+const customConfig = defaultCanaryConfig
+customConfig.gil.bloom = {
```

Comment:
Not sure this was the intention but, this is effectively mutating `defaultCanaryConfig` instead of creating a new config based on it. One alternative would be to create a copy using destructuring:

```suggestion
const customGilConfig = {   
  ...defaultCanaryConfig.gil, 
  bloom: {
    kernelSize: 1,
    luminanceThreshold: 0.1,
    luminanceSmoothing: 0.05,
    intensity: 0.1
  }
}
```

Reply:
(don't apply this suggestion through GitHub because I've selected only lines 15-16 :hankey:)